accountClient/api.go

```diff
@@ -215,3 +217,26 @@ type InternalSigClientInfoResponse struct {
 	PublicKey  string    `json:"public_key"`
 	SigningKey string    `json:"signing_key"`
 }
+
+// InitUpdateEmailRequest wraps values needed to issue a request to start an email update
+type InitiateUpdateEmailRequest struct {
+	AccountID    string    `json:"account_id"`
+	CurrentEmail string    `json:"current_email"`
+	NewEmail     string    `json:"new_email"`
+	CreatedAt    time.Time `json:"created_at"`
```

Comment:
Should this created at field be specified by the service?

Reply:
Agree, this field doesn't need to be in this request struct, as well as the CoolOffEnd as those fields will be determined by the server.